### ISSUE TITLE
eu.rssw.antlr.proparse.checks.TableUsage false negative

### ISSUE THREAD

**Issue by cverbiest** [Riverside-Software/sonar-openedge#1048]
In our application we have a number of shared buffers I want to get rid of.
I created a rule based on the eu.rssw.antlr.proparse.checks.TableUsage template, 
`OpenEdge plugin version: 2.20.0-d175c8d5`

![image](https://github.com/Riverside-Software/sonar-openedge/assets/1312691/69d2a802-044f-45f4-8582-6e11d1917914)

The rule does not detect
```
def shared buffer ALT-XUS for XUSER.
```

It does detect
```
def shared buffer ALT-XUS for XUSER.
find first ALT-XUS.
```

It may be debatable wether defining a buffer should count as usage, but if we remove the shared buffer from the application, this code will fail.
Also, if the same template was used to detect sources containing a table we'd like to remove from the database, sourcen containing only a buffer definition would go undetected, but they would still cause compile failures once the table was actually dropped from the database. 

## Enhancement request

* Consider defining a buffer as "usage" for this rule

* a second minor request, would it be possible to add %3$ to the message table to include the buffer name in the message, currently the rule displays  `Legacy shared buffer XUSER is used`
I'd like it to display  `Legacy shared buffer ALT-XUS is used`

**Comment by gquerret** [comment]
This seems doable

**Comment by gquerret** [comment]
Merged in main
